gradle/modules.properties

```diff
@@ -3,3 +3,4 @@ io.helidon.webserver=io.helidon.webserver:helidon-webserver
 io.helidon.webserver.grpc=io.helidon.webserver:helidon-webserver-grpc
 io.helidon.webserver.testing.junit5=io.helidon.webserver.testing.junit5:helidon-webserver-testing-junit5
 io.grpc=io.grpc:grpc-stub
+grpc.protobuf=io.grpc:grpc-protobuf:1.20.0
```

Comment:
Usually the version information is in `settings.gradle.kts`. Although TBH, I don't like that we have 3 files (this `module-info.java`, `modules.properties`, `settings.gradle.kts`) that need information for the module-to-maven mapping. But for now, that is the pattern we're using.